### ISSUE TITLE
✨ Fix WCAG Issues and create new report

### ISSUE THREAD

**Issue by Findeton** [sequentech/meta#550]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.

### Suggestion

Test WCAG using: https://demo.gremienwahlen.de/booth/40/demo-vote  https://10xx.sequentech.io/

Generate report that is itself accessible.

### Tasks

```[tasklist]
# Tasks
- [x] Deploy 10xx
- [x] Investigate how to generate accessible report
- [x] Generate accessible report
- [x] no skip links at the top of the page
- [x] no regions (landmarks)
- [x] some texts and headings are designed as buttons
- [ ] the form fields (control fields) could possibly also use a fieldset with a legend
- [ ] Check ballots. I thought this would check whether I had given the correct number of votes. Instead, I get an error message saying that this would delete my choice
- [ ] The control fields for the voices are shown to me twice and three times in the reading mode of the Screenreader with a screen reader
- [ ] it's hard to jump from candidate to candidate using TAB
- [ ] the message that I am aware of the number of possible applicants does not tell  me whether I have cast my vote or whether this was prevented
- [x] When error messages appear, they are announced, but I cannot recognize the confirmation windows etc. with a screen reader.
- [ ] at the bottom of the ballot paper it takes me to past error messages. Not usable
- [x] after voting, it goes to election chooser but it doesn't say if i've voted.
```

```[tasklist]
# 10.1.x PRs
- [ ] https://github.com/sequentech/voting-booth/pull/427
- [ ] https://github.com/sequentech/common-ui/pull/379
- [ ] https://github.com/sequentech/election-portal/pull/231
```

```[tasklist]
# Master PRs
- [ ] https://github.com/sequentech/voting-booth/pull/431
- [ ] https://github.com/sequentech/election-portal/pull/232
- [ ] https://github.com/sequentech/common-ui/pull/382
```
